molecule/driver/ec2.py

```diff
@@ -175,6 +224,20 @@ def _get_instance_config(self, instance_name):
             item for item in instance_config_dict if item['instance'] == instance_name
         )
 
+    def _get_windows_instance_pass(self, instance_id, key_file):
```

Comment:
Should we be adding `boto3` / `cryptography` as dependencies in the `ec2` extra? I know the ones who don't use Windows instances don't need it though. Perhaps we can document this instead of having the user run into an error like this and then have to install dependencies.

Reply:
I added a note about the dependencies in the docstring.

Before that I had started to write up some notes on how they're already _somewhat_ dependencies today. Noting here for posterity & prompting future discussion:

* cryptography is already a a transitive molecule dependency (molecule -> ansible -> cryptography). Since there's no guarantee Ansible won't drop it in the future, the try/catch on import seems like a good compromise/safeguard to make sure no future user gets a stacktrace on it if Ansible changes its deps.

* boto3 (or boto if using legacy modules) are required by the ansible modules that actually create the instances, so it's already an implicit dependency of the ec2 driver. As with the cryptography import, having a try/catch on it should keep the user experience on it predictable (e.g. no early stacktraces if operating in some new half-setup venv)